src/HtmlUtils.js

```diff
@@ -208,17 +210,25 @@ module.exports = {
             finally {
                 delete sanitizeHtmlParams.textFilter;
             }
-            return <span className="markdown-body" dangerouslySetInnerHTML={{ __html: safeBody }} />;
         } else {
-            safeBody = content.body;
+            safeBody = _.escape(content.body);
```

Comment:
We're setting `safeBody` as HTML (so we can render emojis as images), but it is arbitrary content (since this was originally plain text) so it needs to be escaped. Lodash is *generally* useful and contains quite a few similar utilities, but yeah, we can probably drop the dependency.